src/test/java/com/vaadin/flow/ai/formfiller/views/FormFillerTextDemo.java

```diff
@@ -216,8 +216,8 @@ public FormFillerTextDemo() {
         ComboBox<String> texts = new ComboBox<>("Select a text or just type your own <br>in the debug Input Source field");
         texts.setItems("Text1", "Text2", "Text3");
         texts.setValue("Text1");
-        texts.setAllowCustomValue(false);
         debugTool.getDebugInput().setValue(getExampleTexts().get("Text1"));
+        texts.setAllowCustomValue(false);
```

Comment:
Why is this needed?

Reply:
Thanks for noticing it! 👍 

This is a mistake from me, I used it for testing the use cases, and then with my last commit, I cleaned up the testing code. 

But it looks like I added to the 220. line rather than the 219. line. I am fixing this!